File: gnsdk-3.08.6.5437o-20170914/wrappers/gnsdk_csharp/src_wrapper/GnPlaylistResults.cs

namespace GracenoteSDK {

/**
* <b>Experimental</b>: GnPlaylistResults
*/
public class GnPlaylistResults : global::System.IDisposable {
  private global::System.Runtime.InteropServices.HandleRef swigCPtr;
  protected bool swigCMemOwn;

  internal GnPlaylistResults(global::System.IntPtr cPtr, bool cMemoryOwn) {
    swigCMemOwn = cMemoryOwn;
    swigCPtr = new global::System.Runtime.InteropServices.HandleRef(this, cPtr);
  }

  internal static global::System.Runtime.InteropServices.HandleRef getCPtr(GnPlaylistResults obj) {
    return (obj == null) ? new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero) : obj.swigCPtr;
  }

  ~GnPlaylistResults() {
    Dispose();
  }

  public virtual void Dispose() {
    lock(this) {
      if (swigCPtr.Handle != global::System.IntPtr.Zero) {
        if (swigCMemOwn) {
          swigCMemOwn = false;
          gnsdk_csharp_marshalPINVOKE.delete_GnPlaylistResults(swigCPtr);
        }
        swigCPtr = new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero);
      }
      global::System.GC.SuppressFinalize(this);
    }
  }

  public GnPlaylistResults() : this(gnsdk_csharp_marshalPINVOKE.new_GnPlaylistResults(), true) {
  }

  public GnPlaylistResultIdentEnumerable Identifiers() {
    GnPlaylistResultIdentEnumerable ret = new GnPlaylistResultIdentEnumerable(gnsdk_csharp_marshalPINVOKE.GnPlaylistResults_Identifiers(swigCPtr), true);
    return ret;
  }

}

}

File: gnsdk-3.08.6.5437o-20170914/wrappers/gnsdk_java/src_wrapper/j2se/com/gracenote/gnsdk/GnPlaylistResults.java

package com.gracenote.gnsdk;

/** 
* <b>Experimental</b>: {@link GnPlaylistResults} 
*/ 
 
public class GnPlaylistResults {
  private transient long swigCPtr;
  protected transient boolean swigCMemOwn;

  protected GnPlaylistResults(long cPtr, boolean cMemoryOwn) {
    swigCMemOwn = cMemoryOwn;
    swigCPtr = cPtr;
  }

  protected static long getCPtr(GnPlaylistResults obj) {
    return (obj == null) ? 0 : obj.swigCPtr;
  }

  protected void finalize() {
    delete();
  }

  public synchronized void delete() {
    if (swigCPtr != 0) {
      if (swigCMemOwn) {
        swigCMemOwn = false;
        gnsdk_javaJNI.delete_GnPlaylistResults(swigCPtr);
      }
      swigCPtr = 0;
    }
  }

  public GnPlaylistResults() {
    this(gnsdk_javaJNI.new_GnPlaylistResults(), true);
  }

  public GnPlaylistResultIdentIterable identifiers() {
    return new GnPlaylistResultIdentIterable(gnsdk_javaJNI.GnPlaylistResults_identifiers(swigCPtr, this), true);
  }

}
